tests/unit/serialization/test_numpy_records.py

```diff
@@ -100,3 +101,39 @@ def test_can_convert_to_records_without_objects_returns_true_otherwise(fast_seri
             store._to_records.assert_called_once_with(sentinel.df)
     finally:
         anr._FAST_CHECK_DF_SERIALIZABLE = orig_config
+
+
+@pytest.mark.parametrize("fast_serializable_check", (False, True))
+def test_can_convert_to_records_mixed_object_column_string_nan(fast_serializable_check):
+    orig_config = anr._FAST_CHECK_DF_SERIALIZABLE
```

Comment:
Is it possible to move all this boilerplate up into the annotation processing, suite generation?  Avoid copying and pasting?

Reply:
refactored, looks cleaner now, thanks